### ISSUE TITLE
Critical Issue, bad perfomance: active PHP session was detected, cause plugin unyson

### ISSUE THREAD

**Issue by otakuyose** [ThemeFuse/Unyson#4042]
An active PHP session was detected
::Rendimiento::
A PHP session was created by a session_start() function call. This interferes with REST API and loopback requests. The session should be closed by session_write_close() before making any HTTP requests.

Cause:
[27-Aug-2020 02:44:06 UTC] PHP Notice: Trying to access array offset on value of type null in /home1/sugoicom/public_html/wp-content/plugins/unyson/framework/helpers/class-fw-resize.php on line 35

my web site is bad performance, is too slow

**Comment by etwordpress01** [comment]
To disable session you can try to use below filter, but i am not sure if this can create an issue with plugin sessions

if (!function_exists('_disable_fw_use_sessions')) {
	add_filter('fw_use_sessions','_disable_fw_use_sessions');
	function _disable_fw_use_sessions(){
		return false;
	}
}

**Comment by otakuyose** [comment]
That code is entered in wp-config, right?

**Comment by etwordpress01** [comment]
No, This is a filter and you can add it in the theme functions.php file

**Comment by sinakheyrandish** [comment]
same here

**Comment by jeremyJJB** [comment]
This definitely worked, my site loads much faster now. The site is running WP 5.5.1  and Unyson 2.7.24

**Comment by markarim** [comment]
Thx, it worked, and my site loads faster.
WP 5.5.1
PHP 7.3

**Comment by corzel** [comment]
I fixed it adding: 
**session_write_close();** 
in lines 231 and 262 in this file: wp-content/plugins/unyson/framework/includes/hooks.php
```
229: session_start();
230: }
231: session_write_close();
232: }

**Comment by sergionline** [comment]
> I fixed it adding:
> **session_write_close();**
> in lines 231 and 262 in this file: wp-content/plugins/unyson/framework/includes/hooks.php
> 
> ```
> 229: session_start();
> 230: }
> 231: session_write_close();
> 232: }
> ```

Thx Thx Thz You are the best!!!!
